keyboards/iris/keymaps/broswen/config.h

```diff
@@ -0,0 +1,62 @@
+/*
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
+
+/* Use I2C or Serial, not both */
+
+#define USE_SERIAL
+// #define USE_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+#undef RGBLED_NUM
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 12
+#define RGBLIGHT_HUE_STEP 5
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#define C6_AUDIO
+
+#define NO_MUSIC_MODE
+
+#ifdef AUDIO_ENABLE
+  #define STARTUP_SONG SONG(NO_SOUND)
+#endif
+
+
+#if !defined(NO_DEBUG) && !defined(CONSOLE_ENABLE)
```

Comment:
Actually, these two blocks are needed.

Somebody pointed that out to me, and I checked.... yeah, both debugging and printing is automatically disabled if the console isn't enabled.